code/game/gamemodes/clock_cult/clock_items/clockwork_armor.dm

```diff
@@ -10,12 +10,25 @@
 
 /obj/item/clothing/head/helmet/clockwork/New()
 	..()
+	ratvar_act()
 	all_clockwork_objects += src
 
 /obj/item/clothing/head/helmet/clockwork/Destroy()
 	all_clockwork_objects -= src
 	return ..()
 
+/obj/item/clothing/head/helmet/clockwork/ratvar_act()
+	if(ratvar_awakens)
+		armor = list(melee = 100, bullet = 100, laser = 100, energy = 100, bomb = 100, bio = 100, rad = 100, fire = 100, acid = 100)
```

Comment:
To clarify what this actually does: 90% damage resistance to melee and projectile attacks, immunity to stun projectiles, reduced damage against bombs (and not being gibbed). If you want to make it the ultimate armor, then have it `return FALSE` on `hit_reaction()' during `ratvar_awakens`.